File: firebird/java/delete/firebird_delete.java
// ----------------------------------------------------------------------
/*
	java/delete/firebird_delete.java

				Jun/01/2011


*/
// ----------------------------------------------------------------------
import java.sql.DriverManager;
import java.sql.Connection;


// ----------------------------------------------------------------------
public class firebird_delete
{

// ----------------------------------------------------------------------
public static void main (String[] args) throws Exception
{
	System.out.println ("*** 開始 ***");

	String	id = args[0];
	System.out.println ("\tid = " + id);

	DriverManager.registerDriver(new org.firebirdsql.jdbc.FBDriver());

	String user     = "sysdba";
	String password = "tiger";
	String database = "/var/tmp/firebird/cities.fdb";

	Connection conn = DriverManager.getConnection
		("jdbc:firebirdsql:localhost/3050:" +
		database, user, password);


	rdb_common.delete_proc	(conn,id);

	rdb_common.display_proc	(conn);

	conn.close ();

	System.out.println ("*** 終了 ***");
}

// ----------------------------------------------------------------------
}
// ----------------------------------------------------------------------

File: firebird/scala/delete/firebird_delete.scala
// -----------------------------------------------------------------
//	delete/firebird_delete.scala
//
//						Aug/25/2015
// -----------------------------------------------------------------
import scala.io.Source
import java.sql._

// -----------------------------------------------------------------
object firebird_delete
{
// -----------------------------------------------------------------
def main (args: scala.Array[String])
{
	println	("*** 開始 ***")

	val id_in = args(0)

	println (id_in)

	firebird_delete_proc (id_in)

	println	("*** 終了 ***")
}

// -----------------------------------------------------------------
def firebird_delete_proc (id_in:String)
{
	val database = "/var/tmp/firebird/cities.fdb"
//	val str_connect= "jdbc:firebirdsql:localhost/3050:" + database
	val str_connect= "jdbc:firebirdsql:cddn007-ee:" + database

	Class.forName ("org.firebirdsql.jdbc.FBDriver")

	val conn = DriverManager.getConnection (str_connect,"sysdba","tiger")

	sql_manipulate.delete_proc (conn,id_in)

	conn.close
}

// -----------------------------------------------------------------
}
// -----------------------------------------------------------------
